gateway/ln-gateway/src/lib.rs

```diff
@@ -852,49 +851,16 @@ impl Gateway {
     /// Requests the gateway to pay an outgoing LN invoice on behalf of a
     /// Fedimint client. Returns the payment hash's preimage on success.
     async fn handle_pay_invoice_msg(&self, payload: PayInvoicePayload) -> Result<Preimage> {
```

Comment:
Note for reviewers: Searching for usage of `PAY_INVOICE_ENDPOINT` you can see it is only used in defining the handler, so `handle_pay_invoice_msg` isn't actually being used anywhere. In this PR we're simply repurposing it and using `PAY_INVOICE_ENDPOINT` on the client side for paying normal invoices.